styles/styles.css

```diff
@@ -77,6 +77,14 @@
     @apply text-light !important;
   }
 
+  .hover-light {
+    @apply hover:bg-light hover:text-dark !important;
+  }
+
+  .hover-dark {
+    @apply hover:bg-dark hover:text-light !important;
+  }
```

Comment:
It's best to avoid using custom classes and to instead use tailwind directly.

If we want to wrap up common lists of classes in a composable / reusable way, we can do that using simple strings containing class lists or better yet, components that encapsulate common styles.

If we want to avoid shipping html with lots of duplicated classes (which is very rarely an actual issue in practice due to gzipping) then there are a few approaches to optimizations, but I don't think that polluting the global class namespace is the right solve.

Reply:
Also worth noting in case you aren't aware that [`!important` can be applied by prepending a `!` to a tailwind class](https://tailwindcss.com/docs/configuration#important-modifier) name, i.e. `!hover:bg-dark`, apologies if you already knew